services/src/main/scala/cromwell/services/womtool/Describer.scala

```diff
@@ -41,21 +42,24 @@ object Describer {
   }
 
   // By this point there are no "out of band" errors that can occur (i.e. those that would indicate a BadRequest, versus just showing up in the `errors` list)
-  private def describeWorkflowInner(factory: LanguageFactory, workflow: WorkflowSource, workflowSourceFilesCollection: WorkflowSourceFilesCollection): WorkflowDescription = {
+  private def describeWorkflowInner(factory: LanguageFactory,
+                                    workflow: WorkflowSource,
+                                    workflowSourceFilesCollection: WorkflowSourceFilesCollection,
+                                    rootWfResolvedImports: RootWorkflowResolvedImports): WorkflowDescription = {
 
     // Mirror of the inputs/no inputs fork in womtool.validate.Validate
     if (workflowSourceFilesCollection.inputsJson.isEmpty) {
       // No inputs: just load up the WomBundle
-      factory.getWomBundle(workflow, "{}", List(HttpResolver(None, Map.empty)), List(factory)) match {
-        case Right(bundle: WomBundle) =>
+      factory.getWomBundleWithImports(workflow, "{}", List(HttpResolver(rootWfResolvedImports, None, Map.empty)), List(factory)) match {
+        case Right((bundle: WomBundle, _)) =>
           WorkflowDescription.fromBundle(bundle, factory.languageName, factory.languageVersionName, List.empty)
         case Left(workflowErrors) =>
           WorkflowDescription.withErrors(workflowErrors.toList)
       }
     } else {
       // Inputs: load up the WomBundle and then try creating an executable with WomBundle + inputs
-      factory.getWomBundle(workflow, "{}", List(HttpResolver(None, Map.empty)), List(factory)) match {
-        case Right(bundle) =>
+      factory.getWomBundleWithImports(workflow, "{}", List(HttpResolver(rootWfResolvedImports, None, Map.empty)), List(factory)) match {
+        case Right((bundle, _)) =>
```

Comment:
TOL (a tiny bit of feature creep...): Since you already have the imports available to you here, could we perhaps modify the `WorkflowDescription.fromBundle` method to also add the full import paths to the description? That way tools like rawls and agora could use the `describe` endpoint to see all of the import resolutions too!